src/mixins.coffee

```diff
@@ -64,3 +64,75 @@ Ember.Widgets.BodyEventListener = Ember.Mixin.create
   _removeDocumentHandlers: ->
     $(@get('bodyElementSelector')).off "click", @_clickHandler
     @_clickHandler = null
+
+Ember.Widgets.TabbableModal = Ember.Mixin.create
+  currentFocus: null
+
+  _focusTabbable: ->
+     # Set focus to the first match:
```

Comment:
This comment makes sense to me, but I find the logic below very confusing. What if you made a function `setFocusToElement(domElement)`, then build an `if...else` block to mirror your comment. So for example: `if @get 'currentFocus'; setFocusToElement(...); else if @$('[autofocus']) is not null; setFocusToElement(...)`. Does that work?